### PR TITLE
GRVDEV-113 Create Arctic Wolf and VPC flow log module

### PR DESCRIPTION
**Pull request by TysonBeck**

### CODE REVIEW

File: aws/arctic-wolf/arctic_wolf.tf [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  description = ""
+}
+
+variable "vpc_flow_log_group_name" {
+  type        = string
+  default     = null
+  description = ""
+}
+
+variable "cloudtrail" {
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: Is this a name, id, arn? Add a suffix to make the var more descriptive.

File: aws/arctic-wolf/arctic_wolf.tf [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  depends_on = [
+    aws_cloudformation_stack.cloudtrail[0],
+  ]
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Outputs
+# ----------------------------------------------------------------------------------------------------------------------
+
+output "cloudtrail" {
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: Check suffix.

File: aws/arctic-wolf/arctic_wolf.tf [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  value       = var.cloudtrail == null ? aws_cloudtrail.default[0].name : null
+  description = ""
+}
+
+output "topicArn" {
```
Comment (jeremyodle) [inline aws/arctic-wolf/arctic_wolf.tf]: Should this be snake case?

**Review by jeremyodle — CHANGES_REQUESTED** [review]

File: aws/arctic-wolf/arctic_wolf.tf (hunk 3 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: Modified name to correctly identify output value

File: aws/arctic-wolf/arctic_wolf.tf (hunk 1 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: changed to cloudtrail_name and all subsequent references

File: aws/arctic-wolf/arctic_wolf.tf (hunk 2 above)
Comment (TysonBeck) [reply aws/arctic-wolf/arctic_wolf.tf]: modified to cloudtrail_name

**Review by jeremyodle — APPROVED** [review]

**Pull request merged**